grails-app/services/org/pih/warehouse/dashboard/IndicatorDataService.groovy

```diff
@@ -1184,15 +1185,16 @@ class IndicatorDataService {
             SELECT p.product_code,
             COUNT(DISTINCT backdated_transaction.shipment_number) as products,
             GROUP_CONCAT(DISTINCT backdated_transaction.shipment_number SEPARATOR ' ') as shipments,
-            DATE_FORMAT(stock_count.last_stock_count, "%d-%M-%Y") FROM (
-            (
-                SELECT t.id as transaction_id, s.shipment_number FROM shipment s
+            DATE_FORMAT(stock_count.last_stock_count, "%d-%b-%Y"),
```

Comment:
I am assuming that we are leaving this as is for now, but I think this will have to be covered as a part of localized date tickets in the near future.